app/Http/Controllers/BarbersController.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Barber;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Gate;
+
+class BarbersController extends Controller
+{
+    public function store(Request $request)
+    {
+        Barber::create($request->all());
+
+        return redirect()->route('barbershop.register');
```

Comment:
I have a gut sense you won't want it to redirect to barbershop create for long, since you may want to create more than one barber at the same barbershop... I'm sure this is fine for now, but just keep in mind how that may change your flows. :)

Reply:
Also: Usually the resource is plural in route name groups. So `barbershops.register`, or, if you agree with what I said above, `barbershops.create`.